packages/services/f-analytics/src/store/analytics.module.js

```diff
@@ -35,30 +43,64 @@ export default {
             requestId: '',
             orientation: '',
             display: ''
-        }
+        },
+        events: []
     }),
 
     actions: {
-        updatePlatformData: ({ commit }, platformData) => {
-            commit('pushPlatformData', platformData);
+        [PUSH_PLATFORM_DATA]: ({ commit, state }, platformData) => {
```

Comment:
Actions shouldn't use `mutation-types` values. And as far as I know actions should be responsible only for manipulation the data and calling the mutation. So we need to move datalayer push from it. I would call the action `updatePlatformData` as action won't be responsible for a push.

```suggestion
        updatePlatformData: ({ commit, state }, platformData) => {
```